pipeline_dp/dataset_histograms/computing_histograms.py

```diff
@@ -403,7 +502,7 @@ def _compute_linf_contributions_histogram_on_preaggregated_data(
     """
     linf = backend.map_tuple(
         col,
-        lambda _, x: x[0],  # x is (count, sum, n_partitions)
+        lambda _, x: x[0],  # x is (count, sum, n_partitions, n_contributions)
```

Comment:
Here I am confused. In some places it is written `x is (count, sum, n_partitions, n_contributions)`, in some places `n_contributions` is omitted.

Reply:
There are more places, I will update them too.

Reply:
updated.